Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR.materialtype

```diff
@@ -0,0 +1,75 @@
+{
+    "description": "Material Type with properties used to define Base PBR, a metallic-roughness Physically-Based Rendering (PBR) material shading model.",
+    "version": 1,
```

Comment:
super nit - simpler description such as the following:
'PBR Material type properties based on metallic-roughness shading model'